package.json

```diff
@@ -119,6 +119,11 @@
         "title": "New Component",
         "category": "OpenShift"
       },
+      {
```

Comment:
There is no need to declare new command, because we can use the same one in tree and palette.